service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2565,7 +2690,7 @@ components:
 
     SystemVersion:
       type: object
-      required: [gitTag, gitHash, github, build]
+      required: [gitTag, gitHash, gitHub, build]
```

Comment:
why this change? this entry in the `required` field no longer matches the actual `SystemVersion.properties.github` field